snapshot/lib/assets/SnapshotHelper.swift

```diff
@@ -7,7 +7,7 @@
 //
 
 // -----------------------------------------------------
-// IMPORTANT: When modifying this file, make sure to 
```

Comment:
With this file being modified, will we be able to support both Xcode 8 and Xcode 9 in parallel?